boards/aithinker/ai_wb2_12f/support/openocd.cfg

```diff
@@ -0,0 +1,5 @@
+# For WCH linkE in DAP mode
```

Comment:
WCH?

Reply:
Yes, WCH, more convenient than the RVDebugger (bl702) or ftdi 232hq.